README.md

```diff
@@ -39,7 +39,7 @@ To make changes in a workspace, first `cd` into the workspace's directory and ru
 You can **make a plan** to find out how your changes will affect the current state of the system:
 
 ```sh
-make plan
+terraform plan
```

Comment:
As noted above, we run this from with Terraform's CLI from individual workspace directories now!